### DISCUSSION TITLE
speed difference between sd/xl/flux/all presets

### DISCUSSION THREAD

**Discussion by omlove28** [lllyasviel/stable-diffusion-webui-forge#1723] [General]
My previous post was deleted so I am re-writing here... Sorry to bother you guys and I still admire lllyasviel so much.

Is there an internal architecture difference between memory allocation or something when I switch from sd preset at the top to xl/flux? I noticed the clip_skip tab is gone but gpu_weights tab is added.

I run the same parameters multiple times and the sd preset ended up 7s while the xl preset resulted in constant slower speed like 10-13s.

Does that mean the sd preset eliminate the gpu_weights function and decided automatically what's the best?

Also how do I get the clip_skip tab back if I'm using xl preset?

Thank you all

**Comment by wizz13150** [comment]
> Is there an internal architecture difference between memory allocation or something when I switch from sd preset at the top to xl/flux? I noticed the clip_skip tab is gone but gpu_weights tab is added.

Sure. The presets use different settings, some settings may only exist in one preset. Like distilled CFG.

> Does that mean the sd preset eliminate the gpu_weights function and decided automatically what's the best?

Obviously.

> Also how do I get the clip_skip tab back if I'm using xl preset?

As usual. Go to the Settings tab. Go to 'User Interface'. Set the clip_skip in the 'Quicksettings list'. Reload the UI.

![image](https://github.com/user-attachments/assets/9e769e93-e695-44e3-9d10-a9ac3b01de7c)

Cheers !

**Reply by omlove28** [reply]
Thank you for response. However the sdxl_clip_l_skip does not work the same as CLIP_stop_at_last_layers. It adds a checkbox and I don't know what it does. There is no slider for 1,2,3, etc

![1](https://github.com/user-attachments/assets/e1ac4862-1739-4392-bec0-e1c3e7f2729b)
![2](https://github.com/user-attachments/assets/1f6967ee-2340-4d1e-83df-2df9b284fa3b)
